apps/system/utils/utils_heapinfo.c

```diff
@@ -196,13 +196,22 @@ static void heapinfo_show_taskinfo(void)
 #endif
 }
 
+static void heapinfo_show_binname(bool init_flag, char *heap_name) {
+	if (init_flag != true) {
+		printf("\n****************************************************************\n");
+		printf("     %s HEAP INFORMATION\n", heap_name);
+		printf("****************************************************************\n");
+	}
+}
```

Comment:
Please ensure that curly braces are placed on the following line.
```suggestion
static void heapinfo_show_binname(bool init_flag, char *heap_name)
{
	if (init_flag != true) {
		printf("\n****************************************************************\n");
		printf("     %s HEAP INFORMATION\n", heap_name);
		printf("****************************************************************\n");
	}
}
```